website/screens/utilities/halstackProvider/examples/customTranslations.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  DxcTextInput,
+  DxcFileInput,
+  HalstackProvider,
+  DxcInset,
+} from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+  const [items, changeItems] = useState(10);
+  const [files, setFiles] = useState([]);
+
+  const itemsPerPageClick = (value) => {
+    changeItems(value);
+  };
+  const callbackFile = (files) => {
+    const updatedFiles = files.map((file) => {
+      return { ...file };
+    });
+    setFiles(updatedFiles);
+  };
+
+  const labels = {
+    formFields: {
+      optionalLabel: "(Optionnel)",
+    },
+    fileInput: {
+      fileSizeLessThanErrorMessage:
+        "La taille du fichier doit être inférieure à la taille maximale.",
+    },
+  };
+
+  return (
+    <HalstackProvider labels={labels}>
+      <DxcInset space="2rem">
+        <DxcTextInput
```

Comment:
I think we should add a `DxcFlex` wrapping both the text input and the file input and add a gap between them.

Reply:
Also, please remove the empty spaces between both inputs.

Reply:
Done.